addons/account/models/ir_module.py

```diff
@@ -72,4 +72,4 @@ def write(self, vals):
     def _load_module_terms(self, modules, langs, overwrite=False):
         super()._load_module_terms(modules, langs, overwrite)
         if 'account' in modules:
-            self.env['account.chart.template']._load_translations(langs=langs)
+            self.env.registry._update_account_translations = lambda env: env['account.chart.template']._load_translations(langs=langs)
```

Comment:
This function can be called by the wizards that install or update langs.
In this case, the registry is already loaded and the `_registry_hook` won't be called.
```suggestion
            if self.env.registry.loaded:
                self.env['account.chart.template']._load_translations(langs=langs)
            else:
                self.env.registry._update_account_translations = lambda env: env['account.chart.template']._load_translations(langs=langs)
```